content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -187,28 +181,27 @@ The following file is an Ingress resource that sends traffic to your Service via
 
 1. Add the following line to the bottom of the `/etc/hosts` file. 
 
-   ```shell
-   172.17.0.15 hello-world.info
-   ```
+    ```shell
```

Comment:
This isn't a shell command (was also wrong in the original).

Reply:
Removed syntax according to https://github.com/kubernetes/website/pull/12783#issuecomment-467223409